Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -97,64 +97,61 @@ private void InitializeInternal()
             bool addedComponents = false;
 
 #if UNITY_EDITOR
-
-            var eventSystems = UnityEngine.Object.FindObjectsOfType<EventSystem>();
-            var standaloneInputModules = UnityEngine.Object.FindObjectsOfType<StandaloneInputModule>();
-
-            CameraCache.Main.transform.position = Vector3.zero;
-            CameraCache.Main.transform.rotation = Quaternion.identity;
-
-            if (eventSystems.Length == 0)
+            if (!UnityEditor.EditorApplication.isPlaying)
```

Comment:
Does InitializeInternal get called when the editor isn't playing?

Reply:
Where does it get called while the editor isn't playing?

Reply:
> Where does it get called while the editor isn't playing? 

Any time you access the instance.